src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -50,6 +50,12 @@ internal override IOperation VisitNoneOperation(IOperation operation, object arg
             return nodes.SelectAsArray(n => Visit(n));
         }
 
+        private ImmutableArray<(U, T)> VisitArray<U, T>(ImmutableArray<(U, T)> nodes) where T : IOperation
```

Comment:
>U [](start = 72, length = 1)

Please remove type parameter U and replace its references with ISymbol. This way we can be sure that we are not skipping any IOperation nodes during the cloning operation.